.tekton/serverless-bundle-137-pull-request.yaml

```diff
@@ -10,10 +10,10 @@ metadata:
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( files.all.exists(x, x.matches('^olm-catalog/serverless-operator/')) || files.all.exists(x, x.matches('^.tekton/')) )
   creationTimestamp: null
   labels:
-    appstudio.openshift.io/application: serverless-operator-136
-    appstudio.openshift.io/component: serverless-bundle-136
+    appstudio.openshift.io/application: serverless-operator-137
+    appstudio.openshift.io/component: serverless-bundle-137
```

Comment:
I do not fully understand, but renaming pipelines, would that also require fisrt to update the `project.yaml` to `1.37` and related updates?

Reply:
This is some logic in the konflux generator. In case we are on main branch in SO, it takes the latest release branche and increments the minor: https://github.com/openshift-knative/hack/blob/7da3b1134fa6077b5332cf8c37ee2d1a3288eba8/pkg/prowgen/prowgen_konflux.go#L326-L334